ajax/libs/kule.lazy/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "kule.lazy",
+  "title": "Kule Lazy",
+  "version": "3.0.1007beta",
+  "description": "Kule CSS Lazy, Do less, but changeable.",
+  "keywords": [
+    "Kule",
+    "Lazy",
+    "Kei",
+    "css",
+    "framework",
+    "components",
+    "effects",
+    "animates",
+    "grid",
+    "mobile",
+    "tablet",
+    "desktop",
+    "rwd",
+    "responsive",
+    "browsers"
+  ],
+  "filename": "kule-lazy-full.min.css",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/keicheng/kule.lazy.git"
+  },
+  "author": "Kei Cheng",
+  "license": "MIT",
+  "homepage": "http://www.kule.tw",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/keicheng/kule.lazy.git",
+    "basePath": "",
```

Comment:
I think the "basePath" should point to `"css"` folder.
That your filemap would match to your commit.